test/xQueryXUtils.ts

```diff
@@ -1,66 +1,78 @@
-import { evaluateXPath, evaluateXPathToBoolean, evaluateXPathToNodes } from 'fontoxpath';
-import { parse } from 'fontoxpath/parsing/xPathParser';
-import { parseAst } from '../demo/parseAst';
 import * as chai from 'chai';
-import { sync, slimdom } from 'slimdom-sax-parser';
+import {
+	evaluateXPath,
+	evaluateXPathToBoolean,
+	evaluateXPathToNodes,
+	parseScript,
+	Language
+} from 'fontoxpath';
+import { slimdom, sync } from 'slimdom-sax-parser';
```

Comment:
We should import `slimdom` directly instead of via `slimdom-sax-parser` this will add it typed.

Reply:
yup, but it will break other things, like using the 'proper' serializer in slimdom